shared/chat/conversation/index.desktop.js

```diff
@@ -108,6 +109,67 @@ class Conversation extends Component<Props, State> {
         </Text>
       </Box>
 
+    const loadingLine = this.props.showLoader ? <LoadingLine /> : null
```

Comment:
instead of all these ternary operators inside of the render lets just pull this stuff up. Honestly a bunch of this stuff could be shared by native and desktop...